sedaro/src/sedaro/branch_client.py

```diff
@@ -33,15 +37,32 @@ def __str__(self):
     def __repr__(self):
         return self.__str__()
 
-    def __getattr__(self, block_name: str) -> BlockClassClient:
-        block_class_module = f'{BASE_PACKAGE_NAME}.model.{snake_case(block_name)}'
-        if block_name not in self._block_class_to_block_group_map \
-                or importlib.util.find_spec(block_class_module) is None:
+    def __getattr__(self, block_type: str) -> BlockClassClient:
+        block_snake, block_pascal = get_snake_and_pascal_case(block_type)
+
+        # Valid block class client options that don't have an associated api (won't have a create method)
+        bcc_options_without_api = {'ConOps'}
+        if block_type in bcc_options_without_api:
+            return BlockClassClient(block_pascal, None, self)
+
+        # check if is a valid option for creating a BlockClassClient & get respective api module file
+        block_api_module = import_if_exists(
+            f'{BASE_PACKAGE_NAME}.apis.tags.{block_snake}_api'
+        )
+
+        # Note: use `casefold` due to things like `GpsAlgorithm` vs `GPSAlgorithm`
+        block_options = set(b.casefold() for b in self._block_class_to_block_group_map) | \
+            {'ConstantLoad'.casefold(), 'Surface'.casefold()}
```

Comment:
Added these in here so didn't actually need to add them in the `BlockGroup`s in `app`